inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/projectlist/ProjectsOverviewPage.java

```diff
@@ -202,6 +208,51 @@ protected void onConfigure()
         return projectList;
     }
     
+    private void addLeaveProjectOption(ListItem<Project> aItem)
+    {
+        User user = userRepository.getCurrentUser();
+        Project currentProject = aItem.getModelObject();
+        ConfirmationDialog confirmLeaveDialog = createLeaveProjectDialog(user, currentProject);
+        AjaxLink<Void> leaveProjectLink = createLeaveProjectButton(user, currentProject,
+                confirmLeaveDialog);
+        aItem.add(leaveProjectLink);
+        aItem.add(confirmLeaveDialog);
+    }
+
+    private ConfirmationDialog createLeaveProjectDialog(User user, Project currentProject)
+    {
+        ConfirmationDialog confirmLeaveDialog = new ConfirmationDialog(MID_CONFIRM_LEAVE,
+                new StringResourceModel("LeaveDialog.title", this),
+                new StringResourceModel("LeaveDialog.text", this));
+        confirmLeaveDialog.setConfirmAction((target) -> {
+            projectService.listProjectPermissionLevel(user, currentProject).stream()
+                    .forEach(projectService::removeProjectPermission);
+            setResponsePage(getPage());
+        });
+        return confirmLeaveDialog;
+    }
+
+    private AjaxLink<Void> createLeaveProjectButton(User user, Project currentProject,
```

Comment:
Just a minor thing. In INCepTION, arguments to constructors and methods are prefixed with a, e.g. aVariable. The a stands for argument and is used to differentiate between member variables and arguments. Hence, the member should be called variable.
In Eclipse you can configure this default prefix "a" under Preferences -> Java -> Code Style set the "prefix list" column in the "parameters" row to "a".